src/main/java/net/syscon/elite/web/config/JwkClient.java

```diff
@@ -0,0 +1,53 @@
+package net.syscon.elite.web.config;
+
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.jwk.JWK;
+import com.nimbusds.jose.jwk.JWKSet;
+import com.nimbusds.jose.jwk.RSAKey;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.net.URL;
+import java.security.PublicKey;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+@Slf4j
+public class JwkClient {
```

Comment:
Looks to be in a different coding style to a lot of the previous code. Mainly the lack of final, var and streams.